### PR TITLE
Rockets basic struture

### PR DESCRIPTION
**Pull request by mwenyoa**
### In this PR, 
- I created the basic structure of the Rockets page
- Added visited link as active

### CODE REVIEW

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @mwenyoa,

Alt Text

Good job so far. There are some issues that you still need to work on to go to the next project, but you are almost there! 

## Highlights 👏
✅ Good PR description
✅ Linters are passing
✅ You documented your work professionally
## Required Changes:

 Please check the comments below with [Required] prefix

Optional Suggestions:
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ function App() {
   return (
     <Routes>
       <Route exact path="/" element={<Rockets />} />
+      <Route path="/Rockets" element={<Rockets />} />
```
Comment (tugsem) [inline src/App.js]: Here, you don't need to repeat the line twice. Please remove it to apply DRY rule.

**Comment by mwenyoa** [comment]
@tugsem Thanks for the feedback, let me work on it.

File: src/components/Header.js [hunk 2]
```diff
@@ -9,10 +9,10 @@ const Header = () => (
       <h1>Space Travelers&apos; Hub</h1>
     </div>
     <nav className="d-flex nav-links">
-      <NavLink to="/Rockets">Rockets</NavLink>
-      <NavLink to="/Missions">Missions</NavLink>
+      <NavLink to="/Rockets" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>Rockets</NavLink>
+      <NavLink to="/Missions" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>Missions</NavLink>
       <div className="line" />
-      <NavLink to="/MyProfile">My Profile</NavLink>
+      <NavLink to="/MyProfile" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>My Profile</NavLink>
```
Comment (tugsem) [inline src/components/Header.js]: You don't need to write this complicated lines here. In order to apply KISS rule, kindly style activated links just by adding active class to the CSS.

File: src/index.css [hunk 3]
```diff
@@ -46,3 +46,24 @@ a {
...
+
+.styled-link {
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.no-styled-link {
+  text-decoration: none;
+  font-weight: normal;
+}
```
Comment (tugsem) [inline src/index.css]: Please remove these lines and just add  `.active { 
text-decoration: underline }` in order to style the links in a short and simple way.

**Review by tugsem — COMMENTED** [review]
## STATUS : APPROVED! 🟢
Hi @mwenyoa,
Alt Text
Your project is complete! There is nothing else to say other than... it's time to close it :shipit:
Congratulations! 🎉


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by tugsem — APPROVED** [review]

**Pull request merged**